widget/assets/js/vg-vimeo.min.js

```diff
@@ -1 +1 @@
ES","VG_UTILS","VG_VOLUME_KEY",function(t,n,i,o,a){return{restrict:"A",require:"^videogular",link:function(r,l,u,c){var m,s,d,f=0,p=0,g=!0,v=0,E=e.isDefined(u.vgVimeoFullscreen)?u.vgVimeoFullscreen:"true",y=function(){c.onUpdateTime({target:c.mediaElement[0]})},h=function(){g=!1;var e=new CustomEvent("playing");c.mediaElement[0].dispatchEvent(e),c.setState(i.PLAY)},b=function(){g=!0;var e=new CustomEvent("pause");c.mediaElement[0].dispatchEvent(e),c.setState(i.PAUSE)},w=function(){c.onComplete()},P=function(e){f=e.seconds,p=e.duration,y()},V=function(e){var t=new CustomEvent("waiting");c.mediaElement[0].dispatchEvent(t)};function S(){var e=new CustomEvent("loadedmetadata");c.mediaElement[0].dispatchEvent(e)}function C(l){if(l){var u,C,A,O=(u=l.match(/^.+vimeo.com\/(.*\/)?([^#\?]*)/))?u[2]||u[1]:null;O&&(C=O,C=u[1]?u[1].split("/")[0]+"?h="+u[2]:u[2]+"?",m?A=m.element:(A=document.createElement("iframe"),e.element(c.mediaElement[0]).replaceWith(e.element(A))),A.src="https://player.vimeo.com/video/"+C+"player_id=vimeoplayer&title=0&byline=0&portrait=0&controls=1",A.style.width="100%",A.style.height="100%",r.$eval(E)&&(A.setAttribute("webkitallowfullscreen",""),A.setAttribute("mozallowfullscreen",""),A.setAttribute("allowfullscreen","")),e.element(A.parentNode.parentNode).find("vg-overlay-play").css("height","calc(100% - 50px)"),(m=new Vimeo.Player(A)).ready().then(function(){!function(){if(angular.element("#vg-controls").css("display","none"),angular.element("#vg-overlay-play").css("display","none"),Object.defineProperty(c.mediaElement[0],"currentTime",{get:function(){return f},set:function(e){f=e,m.setCurrentTime(f)},enumerable:!0,configurable:!0}),Object.defineProperty(c.mediaElement[0],"duration",{get:function(){return p},enumerable:!0,configurable:!0}),Object.defineProperty(c.mediaElement[0],"paused",{get:function(){return g},enumerable:!0,configurable:!0}),Object.defineProperty(c.mediaElement[0],"videoWidth",{get:function(){return s},enumerable:!0,configurable:!0}),Object.defineProperty(c.mediaElement[0],"videoHeight",{get:function(){return d},enumerable:!0,configurable:!0}),Object.defineProperty(c.mediaElement[0],"volume",{get:function(){return v},set:function(e){v=e,m.setVolume(v)},enumerable:!0,configurable:!0}),c.mediaElement[0].play=function(){m.play()},c.mediaElement[0].pause=function(){m.pause()},m.getVolume().then(function(e){v=e,c.onVolumeChange()}),m.getCurrentTime().then(function(e){f=e,S()}),m.getDuration().then(function(e){p=e,S()}),y(),o.supportsLocalStorage()&&0<parseFloat(t.localStorage.getItem(a)||"1")&&(c.mediaElement[0].muted=!1),c.currentState!==i.PLAY){var e=new CustomEvent("canplay");c.mediaElement[0].dispatchEvent(e),!0===c.autoPlay&&n(function(){c.play()})}else m.play().then(function(){})}()}),m.on("play",h),m.on("pause",b),m.on("ended",w),m.on("timeupdate",P),m.on("progress",V))}else m&&m.destroy()}r.$watch(function(){return c.sources},function(e,t){e&&0<e.length&&e[0].src?C(e[0].src.toString()):C(null)})}}}])}(angular);
```

Comment:
So what we changed here and why we did this?

Reply:
In this file I added the implementation of unlisted Vimeo videos cuz it wasn't covered

Reply:
And what we have changed? This is impossible to review. Do we have unminified version showing what we changed?

Reply:
so i can send you both codes un-minified if you want.

but i will put here what i added into the code:

(C = u[1] ? u[1].split("/")[0] + "?h=" + u[2] : u[2] + "?"), >> this line for unlisted videos to check if the video id = 15151515 or 7887878/a6fd5dg8

and controls=1 instead of controls=0 to get the controls from Vimeo player

and these 
 angular.element("#vg-controls").css("display", "none"),
 angular.element("#vg-overlay-play").css("display", "none"),
 
 to hide the Videoguler controls
 
 let me know if you need anything else